dynamic_values.unit_tests.tf

```diff
@@ -39,8 +48,10 @@ locals {
 ##############################################################################
 
 locals {
-  assert_rule_exists_in_map     = lookup(module.unit_tests.security_group_rules, "test-rule")
-  assert_rule_has_correct_field = regex("field", module.unit_tests.security_group_rules["test-rule"].field)
+  # tflint-ignore: terraform_unused_declarations
+  assert_rule_exists_in_map = lookup(module.unit_tests.security_group_rules, "test-rule")
+  # tflint-ignore: terraform_unused_declarations
+  assert_rule_has_correct_field = regex("test-rule", module.unit_tests.security_group_rules["test-rule"].name)
```

Comment:
@ocofaigh fyi, I had to change a test here. When I added type to `security_group_rules` variable inside `dynamic_values/variables.tf` then `module.unit_tests.security_group_rules["test-rule"]` didn't have `field` anymore. Type has to be added because of tflint rule. 

`module.unit_tests.security_group_rules` **before** changes
```
{
     test-rule = {
         field = "field"
         name  = "test-rule"
        }
    }
}
 ```
 
 `module.unit_tests.security_group_rules` **after** changes
 ```
 {
     test-rule = {
         icmp = null
         name = "test-rule"
         tcp  = null
         udp  = null
        }
    }
}
 ```